src/swagger.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
I think so, it could go here or at the top of any of the files in the src folder. It'll scan all the files in src for the SWG tags